src/services/EthService/network/providerHandler.ts

```diff
@@ -79,9 +79,27 @@ export class ProviderHandler {
     return this.injectClient((client) => client.getTransactionCount(address));
   }
 
-  /* Tested */
-  public getTransactionByHash(txhash: string): Promise<TransactionResponse> {
-    return this.injectClient((client) => client.getTransaction(txhash));
+  /* Tested - @todo Test useMultipleProviders */
+  public getTransactionByHash(
```

Comment:
Naming

Reply:
I would prefer not renaming stuff in use throughout the app for this.

Reply:
Ok, if you think it's the best way.